superset-frontend/src/explore/components/PropertiesModal/index.tsx

```diff
@@ -237,16 +231,15 @@ export default function PropertiesModal({
             </FormItem>
             <h3 style={{ marginTop: '1em' }}>{t('Access')}</h3>
             <FormItem label={t('Owners')}>
-              <AsyncSelect
-                isMulti
+              <Select
+                ariaLabel={t('Owners')}
```

Comment:
To avoid the `FormItem` `label` and this `ariaLabel` diverging, could be nice to break this out into a `const`.